pkg/sql/opt/exec/explain/emit.go

```diff
@@ -779,9 +779,7 @@ func (e *emitter) emitNodeAttributes(n *Node) error {
 		e.emitLockingPolicyWithPrefix("right ", a.RightLocking)
 
 	case invertedFilterOp:
-		a := n.args.(*invertedFilterArgs)
-		ob.Attr("inverted column", a.Input.Columns()[a.InvColumn].Name)
-		ob.Attr("num spans", len(a.InvFilter.SpansToRead))
+		// No details are encoded for inverted filters.
```

Comment:
Why not?  Seems like this would change normal explain which I would think we'd want to output inverted filter details.

Reply:
I don't know the reasoning behind it, but we don't encode any details besides the type of operator for filters and inverted filters. The plan gist factory code looks like:

```go
func (f *PlanGistFactory) ConstructFilter(
	input exec.Node,
	filter tree.TypedExpr,
	reqOrdering exec.OutputOrdering,
) (exec.Node, error) {
	f.encodeOperator(filterOp)
	node, err := f.wrappedFactory.ConstructFilter(
		input,
		filter,
		reqOrdering,
	)
	return node, err
}

func (f *PlanGistFactory) ConstructInvertedFilter(
	input exec.Node,
	invFilter *inverted.SpanExpression,
	preFiltererExpr tree.TypedExpr,
	preFiltererType *types.T,
	invColumn exec.NodeColumnOrdinal,
) (exec.Node, error) {
	f.encodeOperator(invertedFilterOp)
	node, err := f.wrappedFactory.ConstructInvertedFilter(
		input,
		invFilter,
		preFiltererExpr,
		preFiltererType,
		invColumn,
	)
	return node, err
}
```

I assumed it was to prevent sensitive details, or too-specific of information from being encoded in the plan gist, since it's meant to just contain the general shape of the plan.

Reply:
Nevermind, I see what you mean about this affecting normal explain. I'll fix it.

Reply:
Fixed.